app/javascript/mastodon/features/ui/index.js

```diff
@@ -128,11 +133,18 @@ class UI extends React.PureComponent {
         </ColumnsArea>
       );
     } else {
+      // TODO: Better check & error handling
```

Comment:
Because if I do that I am basically waiting for all three columns to be fetched until I display them. This way as soon as a column is fetched it is displayed. Also, if fetching just one of them fails, the user is still able to interact with the other two.